src/main/java/Duke.java

```diff
@@ -1,10 +1,71 @@
+import duke.*;
+
+import java.nio.file.Path;
+import java.util.*;
```

Comment:
prevent using wildcards for importing

Reply:
I changed them according to coding standards. Thank you for your comment :)